servicetalk-test-resources/src/main/java/io/servicetalk/test/resources/TestUtils.java

```diff
@@ -32,12 +33,23 @@ private TestUtils() {
      * a suppressed error will be captured and surface through a single {@link AssertionError}.
      * @param errors The queue of captured errors.
      */
-    public static void assertNoAsyncErrors(final Queue<Throwable> errors) {
```

Comment:
Regarding my earlier request to consider using `Iterable`. This turns out to not be needed, queue is fine.